operator/src/test/java/org/keycloak/operator/PodTemplateTest.java

```diff
@@ -0,0 +1,224 @@
+package org.keycloak.operator;
+
+import io.fabric8.kubernetes.api.model.IntOrString;
+import io.fabric8.kubernetes.api.model.PodTemplateSpec;
+import io.fabric8.kubernetes.api.model.PodTemplateSpecBuilder;
+import io.fabric8.kubernetes.api.model.ProbeBuilder;
+import io.fabric8.kubernetes.api.model.apps.Deployment;
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+import org.keycloak.operator.v2alpha1.KeycloakDeployment;
+import org.keycloak.operator.v2alpha1.crds.Keycloak;
+import org.keycloak.operator.v2alpha1.crds.KeycloakSpec;
+import org.keycloak.operator.v2alpha1.crds.keycloakspec.Unsupported;
+
+import java.net.URL;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@QuarkusTest
+public class PodTemplateTest {
+
+    Deployment getDeployment(PodTemplateSpec podTemplate) {
+        var config = new Config(){
+            @Override
+            public Keycloak keycloak() {
+                return new Keycloak() {
+                    @Override
+                    public String image() {
+                        return "dummy-image";
+                    }
+                    @Override
+                    public String imagePullPolicy() {
+                        return "Never";
+                    }
+                    @Override
+                    public String initContainerImage() { return "quay.io/keycloak/keycloak-init-container:legacy"; }
+                    @Override
+                    public String initContainerImagePullPolicy() { return "Always"; }
+                };
+            }
+        };
+        var kc = new Keycloak();
+        var spec = new KeycloakSpec();
+        spec.setUnsupported(new Unsupported(podTemplate));
+        kc.setSpec(spec);
+        var deployment = new KeycloakDeployment(null, config, kc, new Deployment());
+        return (Deployment) deployment.getReconciledResource().get();
+    }
+
+    @Test
+    void testEmpty() {
```

Comment:
We're using access modifiers across the code. Please stick to it.

Reply:
Using access modifiers in tests is useless (even more in a module that will not be published as a `jar`), @vmuzikar can I read about this convention anywhere as it doesn't look to be a shared pattern in the [Keycloak codebase](https://github.com/keycloak/keycloak/blob/main/quarkus/tests/integration/src/test/java/org/keycloak/it/cli/HelpCommandTest.java)? Or. am I missing any reasoning?

Reply:
It's not written anywhere yet as we don't officially have code style defined. But the rule of thumb is: let's not introduce new styles, let's try to keep the project consistent instead.

Reply:
Happy to discuss about this detail and happy to eventually submit a PR to remove all the access modifiers in the test scope as they are completely un-necessary noise IMHO.

For now, done.